examples/orientation_parser_planar.html

```diff
@@ -0,0 +1,118 @@
+               smartZoomHeightMax: 100000,
+            });
+            
+            // Input data : a geoJson file, with point features, with orientation specific properties
+            var panoramics = 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/immersive/exampleParis1/panoramicsMetaDataParis.geojson';
+            var layer = {
+                crsOut: view.referenceCrs,
+            };
+            // Input data are given in Lambert93 projection
+            itowns.proj4.defs('EPSG:2154',
+                '+proj=lcc +lat_1=49 +lat_2=44 +lat_0=46.5 +lon_0=3 +x_0=700000 +y_0=6600000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+            
+            // Add a promise to load oriented point datas
+            var promises = [];
+            promises.push(
+                // first parse the JSON format
+                itowns.Fetcher.json(panoramics, {})
+                // then get geographic informations (coordinates) using GeoJson parser
+                .then(function parseGeoJSON(orientations) { return itowns.GeoJsonParser.parse(orientations, layer);})
+            );        
+            
+            // load collada model of a renault trafic
+            promises.push(ThreeLoader.load('Collada',
+             'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/models/collada/renault_trafic_no_texture.dae')
+             .then(function prepareModel(collada) {
+                // prepare collada model base scale and rotation. in this example, we want the car to go straigth.
+                var colladaModel = collada.scene;
+                colladaModel.scale.set(0.5, 0.5, 0.5);
+                colladaModel.rotateZ(-Math.PI / 2 );
+                // put the model in a group, to allow user to do other transformation (rotation..)
+                var group = new itowns.THREE.Group();
+                group.add(colladaModel);
+                return group; 
+             }));
+
+            // wait for all the promises
+            Promise.all(promises).then(function loadFeaturesModel(res) { loadFeatures(res[0], res[1]);});
+            
+            // place a 3d model oriented for each feature
+            function loadFeatures(collection, colladaModel) {
+   
+                for (const feature of collection.features) {                  
+                
+                    // clone the collada model
+                    var model = colladaModel.clone();
+
+                    // get coordinate from point feature
+                    var coord = feature.vertices[0];
+
+                    // set model position to the ground
+                    var position = coord.xyz();
+                    position.z = 0;
+                    model.position.copy(position);
+                                        
+                    // set model orientation
+                    var quat = itowns.OrientationParser.parse(feature.properties);
```

Comment:
why not use directly `this.quaternionFromAttitude(attitude)` ?

Reply:
Yes I could, but I want to show examples that use of the top level function parse, 
same function called in planar and in globe, to show that the same function can handle planar and globe.

Reply:
You use the same function, but you don't use the same count of parameters (in planar or globe examples). So, if the user have this knowledge, he could directly use the right function.